packaging/aznfs/RPM/aznfs.spec

```diff
@@ -66,6 +66,9 @@ chmod 4755 /sbin/mount.aznfs
 mkdir -p /opt/microsoft/aznfs/data
 chmod 0755 /opt/microsoft/aznfs/data
```

Comment:
you may want to protect this folder?

Reply:
What do you mean by protect this folder? You mean set some mode bits using chattr so customers cannot modify this folder?

Reply:
Added TODO comment here as well as in postinst script